code/src/App.js

```diff
@@ -1,9 +1,97 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import Form from 'components/Form.js'
+import Thoughts from 'components/Thoughts.js'
+import Loading from 'components/Loading.js'
 
 export const App = () => {
+  const [newThought, setNewThought] = useState('')
+  const [loading, setLoading] = useState(true)
+  const [thoughts, setThoughts] = useState([])
+  const [countLikes, setCountLikes] = useState(0)
+  const [characters, setCharacters] = useState(0)
+  const [errorMsg, setErrorMsg] = useState(false)
+  const url = 'https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts'
+
+  const fetchThoughts = () => {
+    setLoading(true);
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => setThoughts(data))
+      .catch((error) => console.log(error))
+      .finally(() => { setLoading(false) })
+  }
+  useEffect(() => {
+    fetchThoughts()
+  }, [])
+  console.log(errorMsg)
+  const handleFormSubmit = (event) => {
+    console.log(newThought)
+    event.preventDefault()
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        message: newThought
+      })
+    }
+    if (newThought.length > 4) {
```

Comment:
It seems like this if condition is not fully functional, because a  new thought is still posted if the length = 4.

Reply:
I think that shouldn't be possible because even without adding an if statement the post request blocks it if the text is under 5 characters. The if statement only adds the error msg.